OpenRA.Game/Settings.cs

```diff
@@ -72,7 +72,7 @@ public class ServerSettings
 		[Desc("Disallow AI bots.")]
 		public bool LockBots = false;
 
-		public string TimestampFormat = "HH:mm";
+		public string TimestampFormat = "dd/MM - HH:mm:ss";
```

Comment:
"o" seems to also include the microseconds which might be "too much" again, there shouldn't be many events on the server that requires deeper logging than seconds afaik?